Header header
int32 node_id
geometry_msgs/Pose pose
sensor_msgs/PointCloud2 cloud